generators/kubernetes/templates/registry/_jhipster-registry.yml

```diff
@@ -125,13 +121,13 @@ spec:
             LAST_POD_INDEX=$((${CLUSTER_SIZE} - 1))
             REPLICAS=""
             for i in $(seq 0 $LAST_POD_INDEX); do
-              REPLICAS="${REPLICAS}http://admin:${SECURITY_USER_PASSWORD}@${STATEFULSET_NAME}-${i}.jhipster-registry.${STATEFULSET_NAMESPACE}.svc.cluster.local:8761/eureka/"
+              REPLICAS="${REPLICAS}http://admin:${JHIPSTER_REGISTRY_PASSWORD}@${STATEFULSET_NAME}-${i}.jhipster-registry.${STATEFULSET_NAMESPACE}.svc.cluster.local:8761/eureka/"
               if [ $i -lt $LAST_POD_INDEX ]; then
                 REPLICAS="${REPLICAS},"
               fi
             done
-            export EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=$REPLICAS
-            echo "EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=${REPLICAS}"
+            export EUREKA_CLIENT_SERVICEURL_DEFAULTZONE=$REPLICAS
+            echo "EUREKA_CLIENT_SERVICEURL_DEFAULTZONE=${REPLICAS}"
 
```

Comment:
I didn't build that initialisation block but tried to figure out what was causing my Registries failure  past my last time of successful usage of 2.5.8

If you have a better replica initialization, let me know, please. 

I was testing the current state with deploying up to 50 spring boot microservices under heavy conditions to provoke failure and pick the configuration with the best behavior.

I will check one thing and report again